File: js_projrepo/algo_e.js
var main = function(){
	var a = 13, b=8;
	var result = gcd(a,b);
	console.log("Result is "+result);
}

function gcd(num1,num2){
	
		var rem = num1%num2;
		if(rem == 0){
			return num2;
		}
		else{
			return gcd(num2,rem);
		}

};

if(require.main === module){
	main();
}

File: c_projrepo/algo_E.c
#include<stdio.h>
#include<conio.h>
int gcd(int,int);


int main(){
	int a = 24,b = 36;
	int result = gcd(a,b);
	printf("%d",result);
	getch();
}

int gcd(int num1,int num2){
	int rem = num1%num2;
	if(rem == 0){
		return num2;
	}
	else{
		gcd(num2,rem);
	}
}

